src/Akka.Persistence.Sql/Snapshot/ByteArraySnapshotDao.cs

```diff
@@ -320,30 +320,16 @@ await connection
             }
         }
 
-        // TODO: This should be converted to async
-        public void InitializeTables()
+        public async Task InitializeTables()
         {
-            using var connection = _connectionFactory.GetConnection();
-
-            try
+            await using var connection = _connectionFactory.GetConnection();
+            if (connection.UseDateTime)
             {
-                if (connection.UseDateTime)
-                {
-                    connection.CreateTable<DateTimeSnapshotRow>();
-                }
-                else
-                {
-                    connection.CreateTable<LongSnapshotRow>();
-                }
+                await connection.CreateTableAsync<DateTimeSnapshotRow>(TableOptions.CreateIfNotExists);
```

Comment:
Tell Linq2Db to generate a `IF NOT EXIST` script for the `CREATE TABLE` SQL script